release-notes/release-notes-v0.8.0.0.md

```diff
@@ -0,0 +1,25 @@
+# Pyserini Release Notes (v0.8.0.0)
+
+**Release date: March 12, 2020**
+
++ Exposed different Lucene analyzers in Python with convenient wrappers.
++ Augmented various methods in `IndexReaderUtils` to support different analyzers.
++ Refactored code base to match underlying Anserini changes.
+
```

Comment:
Also according to the merged PR's:
+ Exposes methods for converting between internal Lucene docid and external collection docid
+ Simplify requirements to include only pyjnius

Reply:
Added as requested!